### ISSUE TITLE
Typing in "Default" tab uses synchronous server feedback and becomes unusable on high-latency connections

### ISSUE THREAD

**Issue by p-e-w** [oobabooga/text-generation-webui#5028]
### Describe the bug

Typing in "Default" tab uses synchronous server feedback and becomes unusable on high-latency connections.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

1. Use (or simulate) a high-latency connection (though it's noticeable even on a standard LAN).
2. Go to "Default" tab.
3. Start typing fast in the "Input" field (holding down a key works well to demonstrate the problem).

The input stutters and a significant lag between key presses and characters being displayed can be observed. Depending on the latency, this can range from annoying to making the "Default" tab essentially unusable.

It appears the `change` event handler attached to the textbox that is used to update the token counter introduces the server feedback as a *blocking* part of the frontend event processing. Which is terrible. I've tried to find a way to make Gradio process the event asynchronously, but there doesn't appear to be one. The more I work with Gradio, the more it seems to me that it is deeply flawed from a performance perspective.

### Logs

```shell
None
```

### System Info

```shell
Hardware independent
```

**Comment by Ph0rk0z** [comment]
You sure it's not the browser? The text isn't sent realtime. I use it exclusively over lan and it's a wifi link so it routinely loses connection, especially when my server generates interference.

**Comment by p-e-w** [comment]
@Ph0rk0z 

> The text isn't sent realtime.

Yes it is. The token counter at the bottom right of the input textbox is updated whenever the input changes, via

https://github.com/oobabooga/text-generation-webui/blob/4b25acf58f78ee8821fc5bf325f602583bfa513f/modules/ui_default.py#L99

The token count is computed in Python, on the server, whenever the value of the textbox changes, in other words, while the user is typing, on every keystroke except navigation keys.

Which isn't necessarily a problem in itself. The real problem is that this interferes with typing.
